kernel-modules/build/build-kos

```diff
@@ -68,6 +71,11 @@ build_ko() (
             return 1
         }
 
+        # directory in which the probe source lives.
+        probe_directory="/probe/"
+        # resulting object file from the build
+        collector_probe="/probe/probe.o"
+
```

Comment:
Are these changes still needed? My understanding is that they were here to fallback to the previous version of the probe, but I don't see them being modified anymore.

Reply:
Agreed, they aren't needed any longer. I'll adjust this.